scalecube-configuration/src/main/java/io/scalecube/configuration/repository/couchbase/CouchbaseRepository.java

```diff
@@ -79,37 +97,94 @@ public CouchbaseRepository(AsyncBucket bucket) {
   }
 
   @Override
-  public Flux<Document> readList(String tenant, String repository) {
+  public Flux<Document> readAll(String tenant, String repository, Integer version) {
+    return Flux.from(
+            RxReactiveStreams.toPublisher(
+                bucket.query(
+                    ViewQuery.from("keys", "by_keys").key(tenant + DELIMITER + repository))))
+        .flatMap(asyncViewResult -> RxReactiveStreams.toPublisher(asyncViewResult.rows()))
+        .flatMap(
```

Comment:
Can we reduce flatMap operators here?

Reply:
Fixed

Reply:
Result of prev. flatMap (viewResult) is using in next. (viewRow)